.editorconfig

```diff
@@ -8,4 +8,8 @@ indent_size = 2
 end_of_line = lf
 insert_final_newline = true
 indent_style = tab
-indent_size = 4
\ No newline at end of file
+indent_size = 4
+
+[composer.json]
+indent_style = tab
+indent_size = 4
```

Comment:
For files which is not php/css/js, we should use 2 spaces indent.

Reply:
I simply followed what already existed in another 10up plugin.

Reply:
What is the plugin you refered to?

Reply:
https://github.com/10up/autoshare-for-twitter/blob/develop/composer.json